createdisk.sh

```diff
@@ -74,6 +74,7 @@ EOF
     ${SSH} core@${VM_IP} -- sudo systemctl disable firewalld
     ${SSH} core@${VM_IP} -- cat /etc/microshift/config.yaml.default > config.yaml
     ${YQ} eval --inplace ".dns.baseDomain = \"${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\""  config.yaml
+    ${YQ} eval --inplace ".ingress.listenAddress = null"  config.yaml
```

Comment:
For master and release-4.17 this commit is not required because it is already fixed there.